src/main/resources/application.yml

```
@@ -44,3 +44,18 @@ app:
       - "https://www.howread.net"
       - "https://howread.net"
       - "http://localhost:3000"
+      -
+
+      # S3
+cloud:
+  aws:
+    credentials:
+      accessKey: ${AWS_ACCESS_KEY}
+      secretKey: ${AWS_SECRET_KEY}
+    s3:
+      bucketName: howread-dev-images
```

버킷에 이미지만 저장하는 것이 아니라 다양한 파일 객체를 저장할 수 있는데요, (ex. PDF 등) 그래서 버킷명은 범용적으로 만들면 좋을 것 같습니다.

```suggestion
      bucketName: howread-dev-assets
```

---

넵 알겠습니다!